src/components/CustomEditor/CustomEditor.tsx

```diff
@@ -33,20 +38,51 @@ export const CustomEditor: React.FC<Props> = ({ value, onChange, context, type =
    */
   const templateSrv = getTemplateSrv();
 
+  /**
+   * Theme
+   */
+  const theme = useTheme2();
+
   /**
    * Format On Mount
    */
   const onEditorMount = useCallback(
     (editor: monacoType.editor.IStandaloneCodeEditor) => {
-      if (context.options.editor.format !== Format.AUTO) {
+      if (context.options.editor.format !== Format.AUTO || type === EditorType.STYLES) {
         return;
       }
 
       setTimeout(() => {
         editor.getAction('editor.action.formatDocument').run();
       }, 100);
     },
-    [context.options.editor.format]
+    [context.options.editor.format, type]
+  );
+
+  /**
+   * Add all possible string and number properties from theme
+   */
+  const extractProperties = useCallback(
+    (
+      entries: Array<[string, unknown]>,
+      parentLabel = 'theme'
+    ): Array<{ label: string; detail: string; kind: CodeEditorSuggestionItemKind.Property }> => {
```

Comment:
can be just `CodeEditorSuggestionItem[]`

Reply:
updated